package.json

```diff
@@ -45,7 +46,7 @@
     "@babel/preset-react": "^7.0.0",
     "babel-jest": "^24.9.0",
     "jest": "^24.9.0",
-    "magic-script-components": "2.0.0",
+    "magic-script-components": "git+https://github.com/magic-script/magic-script-components.git#XrClient",
```

Comment:
I'm not sure what the best approach is here. I can't add npm version dependency until we publish, but I'd like things to work out of the box in the mean time.

Reply:
you can't commit it as is.  This needs to be the current version +1.  You'll need to merge to master and publish to NPM in tandem since you're creating a dependency here for this specific feature.  @creationix how would you recommend handling this?

Reply:
The problem is that this whole thing also depends on multi-prism support which we aren't quite ready to publish yet as I understand it (but it's already available on master for magic-script-components-react-native). That makes the dependencies a bit tricky in the mean time.

Reply:
Maybe commit this to a feature branch instead of master?

Reply:
Updated to remove github dependency. I confirmed that this doesn't interfere with getting the sample app to build and run.